webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -0,0 +1,249 @@
+import { sprintf, n__, translate as __ } from '../react_app/common/I18n';
+
+// Array contains list of host ids
+$.cookieName = `_ForemanSelected${window.location.pathname.replace(/\//, '')}`;
```

Comment:
Don't you need to import `$`?

Reply:
thanks, updated, and also moved the deprecated jquery.cookie next to the code.